devnotes/pnor_layout.md

```diff
@@ -0,0 +1,181 @@
+# About this document
+This document gathers known informations about partition layout in the original
```

Comment:
@IgorBagnucki There is no plural from `information`. Please use https://languagetool.org or different tool for improving documentation quality

Reply:
Fixed